src/app/core/services/proof.service.ts

```diff
@@ -13,11 +13,17 @@ import { GeolocationService } from './geolocation.service';
   providedIn: 'root'
 })
 export class ProofService {
+  enableGeolocation: string = 'disable'; 
```

Comment:
```suggestion
  enableGeolocation = 'disable'; 
```
The initialization already make the variable inferenced as string type, so there's no need for explicit type declaration.